fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/cpp/projects/MakefileUtils.java

```diff
@@ -354,10 +354,10 @@ private String dependanciesObject(String classname, String sufix) {
 		return code;
 	}
 	//======================================================
-	private String dependanciesObjectAddFile(String classname, String filename, boolean mainClass) {
+	private String dependanciesObjectAddFile(String classname, String path, String filename, boolean mainClass) {
 		String code = "$(OBJDIR)/" + filename + ".o:";
 		if (mainClass)	//	Do not manage path
```

Comment:
I think we can delete this comment:
//	Do not manage path

Reply:
ok, problem is I don't really understand what case this difference in path management is supposed to cover…